<?xml version="1.0"?>
<launch>
    <node pkg="master_sync_fkie" type="master_sync" name="master_sync" output="screen"/>
    <node pkg="master_discovery_fkie" type="master_discovery" name="master_discovery" output="screen"/>

    <rosparam file="$(find eurobot_loc)/config/main_robot_params.yaml" command="load" ns="main_robot"/>
    <rosparam file="$(find eurobot_stm)/configs/stm.yaml" command="load" ns="main_robot"/>
    <rosparam file="$(find eurobot_main)/config/bt_params.yaml" command="load" ns="main_robot"/>
    <param name = "field/color" value = "orange"/>

    <group ns = "main_robot">
        <include file="$(find eurobot_nav)/launch/motion_planner_launch.launch"/>
        <node name="stm_node" pkg="eurobot_stm" type="STM.py" output="log" respawn="true"/>
        <node pkg="urg_node" name="lidar_node" type="urg_node" args="_ip_address:=192.168.0.10" respawn="true" output="screen">
            <param name="frame_id" value="/main_robot_laser"/>
        </node>
        <node pkg="eurobot_loc" name="particle_filter_node" type="particle_filter_node.py" respawn="true" output="log"/>
        <node pkg="eurobot_stm" name="manipulator_node" type="manipulator.py" respawn="true" output="screen"/>
        <!--node pkg="eurobot_main" name="behavior_tree" type="first_iter.py" respawn="true" output="screen"/-->
    </group>
</launch>